tests/Aspire.Components.Common.Tests/ConformanceTests.cs

```diff
@@ -318,7 +319,9 @@ public async Task HealthCheckReportsExpectedStatus(string? key)
 
         HealthCheckService healthCheckService = host.Services.GetRequiredService<HealthCheckService>();
 
+#pragma warning disable xUnit1030 // Do not call ConfigureAwait(false) in test method
```

Comment:
Curious, is this a new analyzer in v3?

Reply:
It shouldn't be new. No idea why it wasn't showing up before.